x-pack/plugins/apm/public/components/app/correlations/use_latency_correlations.ts

```diff
@@ -86,20 +86,25 @@ export function useLatencyCorrelations() {
       };
 
       // Initial call to fetch the overall distribution for the log-log plot.
-      const { overallHistogram, totalDocCount, percentileThresholdValue } =
-        await callApmApi(
-          'POST /internal/apm/latency/overall_distribution/transactions',
-          {
-            signal: abortCtrl.current.signal,
-            params: {
-              body: {
-                ...fetchParams,
-                percentileThreshold: DEFAULT_PERCENTILE_THRESHOLD,
-                chartType: LatencyDistributionChartType.latencyCorrelations,
-              },
+      const {
+        overallHistogram,
+        totalDocCount,
+        percentileThresholdValue,
+        durationMin,
+        durationMax,
+      } = await callApmApi(
+        'POST /internal/apm/latency/overall_distribution/transactions',
+        {
+          signal: abortCtrl.current.signal,
+          params: {
+            body: {
+              ...fetchParams,
+              percentileThreshold: DEFAULT_PERCENTILE_THRESHOLD,
+              chartType: LatencyDistributionChartType.latencyCorrelations,
```

Comment:
This call is exactly the same as the one on `x-pack/plugins/apm/public/components/app/correlations/use_failed_transactions_correlations.ts` do you think it is worth extracting it and reusing it?

Reply:
Since we'd like to get this as a bugfix into `8.4` I'd like to avoid more refactoring, I added a comment to the meta issue  in the tech debt section instead to follow up on it (`Revisit hooks that fetch correlations results and possibly consolidate duplicate code like fetching the overall histogram.`): https://github.com/elastic/kibana/issues/118009